kanidm_unix_int/Cargo.toml

```diff
@@ -58,6 +58,7 @@ serde_derive = "1.0"
 serde_cbor = "0.11"
 structopt = { version = "0.3", default-features = false }
 
+libsqlite3-sys = { version = "0.17", features = ["bundled"] }
```

Comment:
I'm a bit worried about this, because some distros that end up using this may not like this. Is there a way for us to have a feature flag for if we use bundled libsqlite?

Reply:
Something like
```
[features]
...
default = [ "libsqlite3-sys/bundled" ]
```

should do the job, then you can opt-out with --no-default-features